src/features/swap/helpers/__tests__/utils.test.ts

```diff
@@ -175,4 +181,62 @@ describe('Swap helpers', () => {
       expect(result).toEqual(50)
     })
   })
+
+  describe('isOrderPartiallyFilled', () => {
+    it('returns true if a buy order is partially filled', () => {
+      const mockOrder = {
+        executedBuyAmount: '10',
+        buyAmount: '100000000000000000000', // 100 tokens
+        executedSellAmount: '50000000000000000000', // 50 tokens
+        sellAmount: '100000000000000000000', // 100 tokens
+        kind: 'buy',
+      } as unknown as SwapOrder
+
+      const result = isOrderPartiallyFilled(mockOrder)
+
+      expect(result).toBe(true)
+    })
+
+    it('returns false if a buy order is fully filled', () => {
+      const mockOrder = {
+        executedBuyAmount: '0',
+        buyAmount: '100000000000000000000', // 100 tokens
```

Comment:
Same here, shouldn't `executedBuyAmount` be the same as `buyAmount` for it to be fully filled?